### PR TITLE
Terms of Service for OVS

### LINKED ISSUE #388: Terms of Service
Draft copy at https://docs.google.com/document/d/1vLb3-_NFT6BgbGFdjjRF4NhVqp6Ay2eDSFLbafigjK4/edit#heading=h.9ra1j7m82h9r

Note that the TOS is broken into two sections: a short summary in bullet form and then a longer body of legalese.

### PR DESCRIPTION
**Pull request by mbertrand**
#### What are the relevant tickets?
- Closes #388 

#### What's this PR do?
- Adds a 'Terms of Service' page
- Adds links to the page in the footer and at the bottom of the FAQ.

#### How should this be manually tested?
Check that the above links display and direct to the terms of service page, and that the page accurately displays the terms.

### CODE REVIEW

File: static/js/containers/TermsPage.js [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+                  available. We will do our best to accommodate these requests,
+                  resources permitting.
+                </li>
+                <li>
+                  <span className="terms-strong">
+                    No permanent storage of video.{" "}
+                  </span>
+                  We will store your source video for at least 6 years. Even if
+                  your video is removed from active streaming (see above), it
+                  will be archived for up to 6 years. "You may download the
```
Comment (rhysyngsun) [inline static/js/containers/TermsPage.js]: Either an extraneous or unclosed quote here.

File: static/js/containers/TermsPage.js [hunk 2]
```diff
@@ -0,0 +1,605 @@
...
+              </div>
+              <div className="terms-div">
+                You may remove content you have posted on ODL Video at any time.
+                When you delete content from ODL Video, such deleted content,
+                while not available to the viewing public and other ODL Video
+                users, will remain on the ODL Video server until such time as
+                you make a specific request to ODL Video for permanent deletion
+                of such content from the ODL Video server. Such requests must be
+                made in writing, via email, to ODL Video customer service at the
+                following address: odl-video-support@mit.edu. When you do remove
```
Comment (rhysyngsun) [inline static/js/containers/TermsPage.js]: Could you link this email like you did earlier in this doc?

File: static/js/containers/TermsPage.js [hunk 3]
```diff
@@ -0,0 +1,605 @@
...
+                the ODL Video site is protected by our copyrights, the
+                copyrights of our partners and/or the copyrights of the user who
+                posted such materials. Materials uploaded to ODL Video may be
+                subject to posted limitations on usage, reproduction and/or
+                dissemination. You are responsible for adhering to such
+                limitations if you download the materials.
+              </div>
+              <div className="terms-div">
+                All user-generated content will be uploaded onto the site under
+                a Creative Commons License (see http://www.creativecommons.org/)
```
Comment (rhysyngsun) [inline static/js/containers/TermsPage.js]: Could you make this a link?

File: static/js/containers/TermsPage.js (hunk 1 above)
Comment (mbertrand) [reply static/js/containers/TermsPage.js]: good catch ✅

File: static/js/containers/TermsPage.js (hunk 2 above)
Comment (mbertrand) [reply static/js/containers/TermsPage.js]: ✅

File: static/js/containers/TermsPage.js (hunk 3 above)
Comment (mbertrand) [reply static/js/containers/TermsPage.js]: ✅

**Pull request merged**